### ISSUE TITLE
Sym link?

### ISSUE THREAD

**Issue by briannhinton** [mathiasbynens/dotfiles#516]
In the readme you talk about how you symlink the dot files into your home directory. When I do this I'm unable to get terminal coloring, etc , and a few other things working. How are you symlinking exactly? I've tried a few methods, and the only way I have it working is by directly copy & pasting.

**Comment by GarethLewin** [comment]
The README says:

>You can clone the repository wherever you want. (I like to keep it in ~/Projects/dotfiles, with ~/dotfiles as a symlink.) The bootstrapper script will pull in the latest version and copy the files to your home folder.

So assuming you decide to clone into ~/Projects/dotfiles you would run `ln -s ~/Projects/dotfiles ~/dotfiles" `

**Comment by briannhinton** [comment]
Yeah. I did that. And I had .osx didn't run properly. (terminal colors
didn't install) I assumed that was a path issue. I removed $HOME and that
had zero effect. Quite a few other things didn't work either.

On Thu, Mar 26, 2015 at 3:41 PM, Gareth Lewin <notifications@github.com>
wrote:

> The README says:
>
> You can clone the repository wherever you want. (I like to keep it in
> ~/Projects/dotfiles, with ~/dotfiles as a symlink.) The bootstrapper script
> will pull in the latest version and copy the files to your home folder.
>
> So assuming you decide to clone into ~/Projects/dotfiles you would run ln
> -s ~/Projects/dotfiles ~/dotfiles"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mathiasbynens/dotfiles/issues/516#issuecomment-86687003>
> .
>



-- 
- Brian
________________
